### PR TITLE
Genovation plum 92 teensy lc working

### PR DESCRIPTION
**Pull request by barrar**

### CODE REVIEW

**Comment by jackhumbert** [comment]
Awesome! Is this handwired or with a PCB? We have a directory specifically for handwired projects :)

File: keyboards/genovation_plum_92_teensy_lc/matrix.c [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+}
+
+inline
+matrix_row_t matrix_get_row(uint8_t row)
+{
+    return matrix[row];
+}
+
+// if you run your finger down each row, you can get a keymap!
+void matrix_print(void)
```
Comment (barrar) [inline keyboards/genovation_plum_92_teensy_lc/matrix.c]: Useful for handwired builds or unknown keymaps, making a keymap is very easy

File: keyboards/genovation_plum_92_teensy_lc/genovation_plum_92_teensy_lc.h [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#ifndef GEKNOVATION_PLUM_92_TEENSY_LC_H
+#define GEKNOVATION_PLUM_92_TEENSY_LC_H
+
+#include "quantum.h"
+
+#define kNO KC_NO  //because now everything just aligns magically :P
```
Comment (barrar) [inline keyboards/genovation_plum_92_teensy_lc/genovation_plum_92_teensy_lc.h]: Looks so much cleaner

File: keyboards/genovation_plum_92_teensy_lc/flash.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./teensy_loader_cli --mcu=TEENSYLC -wv ../../genovation_plum_92_teensy_lc_default.hex
```
Comment (barrar) [inline keyboards/genovation_plum_92_teensy_lc/flash.sh]: Better way to do this in Makefile?

File: keyboards/genovation_plum_92_teensy_lc/config.h [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 16
+#define DIODE_DIRECTION ROW2COL
+
+/* define if matrix has ghost */
+#define MATRIX_HAS_GHOST //GROSS, had to solder some wires to make it better
+
+/* Set 0 if debouncing isn't needed */
+//I've used 2ms for a while and had zero problems, but 3ms or 5ms is safer
+#define DEBOUNCE    2
```
Comment (barrar) [inline keyboards/genovation_plum_92_teensy_lc/config.h]: Works fine with 2, with 1 it bounces 5% of the time

**Comment by barrar** [comment]
http://imgur.com/r/MechanicalKeyboards/0RM7Qbj

**Comment by barrar** [comment]
https://deskthority.net/photos-f62/plum-keyboard-usb-t14640.html

**Comment by jackhumbert** [comment]
I don't see an alternate PCB for sale on DT - did you wire the Teensy to the existing matrix?

**Comment by barrar** [comment]
Yes, and I used the existing wires.

**Comment by jackhumbert** [comment]
Cool! Because of the variability in wiring it up, I think this will fit best in the `handwired` folder :)

**Comment by barrar** [comment]
I'm sure people will just wire it the same way I did when they read the instructions. No variability there unless you want to criss cross wires everywhere for no reason.

**Comment by jackhumbert** [comment]
I try not to assume what people will do :) the (possible) variability and the non-straight-forward DIY nature of this project make it a good candidate for the handwired folder. We're thinking about renaming the folder to include projects like this in the name, and reserve top-level folders for full, purchasable kits and keyboards.

File: keyboards/genovation_plum_92_teensy_lc/flash.sh (hunk 3 above)
Comment (fredizzimo) [inline keyboards/genovation_plum_92_teensy_lc/flash.sh]: Yes, the `teensy` target should be moved from `avr.mk` to `common.mk`

And then you make it with `make teensy`

File: keyboards/genovation_plum_92_teensy_lc/instructions.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
+# Teensy LC, 3.0, 3.1, 3.2 support
```
Comment (fredizzimo) [inline keyboards/genovation_plum_92_teensy_lc/instructions.md]: I think you can remove this file.

**Comment by fredizzimo** [comment]
The `teensy_loader_cli` should also be removed. Installing tools are part of the build instructions.

**Comment by fredizzimo** [comment]
Could you also please include a readme file?

**Comment by jackhumbert** [comment]
@barrar I'm gonna close this for now - whenever you get the chance to make the changes mentioned, feel free to open this back up!

**Pull request closed without merging**